app/keyboard.py

```diff
@@ -0,0 +1,112 @@
+from telegram import InlineKeyboardButton, \
+    InlineKeyboardMarkup, ParseMode, KeyboardButton, ReplyKeyboardMarkup, ReplyKeyboardRemove
+
+"""
+This file will create button structures that we will use
```

Comment:
Alike comment should be in the top of the file

Reply:
will be fixed in the next commit